### DISCUSSION TITLE
Notification - sind dort externe Anfragen gesperrt?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1132] [Q&A]
Ich habe versucht in einer Notification eine HTML zur Eingabe einer Zahl zu nutzen.
Die Zahlen können angeklickt werden, man kann sie aber nicht loggen.
`console.log` aus der HTML hat keine Wirkung. Auch ein `fetch()` zum Holen von Daten zeigt kein Ergebnis.
Deswegen ist meine Vermutung, dass dort irgendeine Einschränkung ist?

Oder hat es grundsätzlich mit dem Popup als Solches zutun? Falsche Befehle in der HTML?

Auf der `jarvis-Seite` normal erhalte ich `console.log() Ausgaben` und `fetch` `funktioniert` auch.

Gibt es eine besondere Einstellung, die man dem Popup bei den Notification beachten muss?

`document.getElementById(id)` hat auch keine Wirkung in der Notification, auf der normalen Seite funktioniert es.
Kann mir jemand einen Tipp geben?

**Comment by Zefau** [comment]
Du versuchst javascript zu nutzen, nicht nur HTML. Javascript geht nicht. Im Widget muss ich es extra in iFrame betten, damit das funktioniert. Das ist in der Notification nicht.
